manticore/platforms/windows.py

```diff
@@ -175,14 +175,17 @@ def __init__(self, path, additional_context = None, snapshot_folder=None):
         assert nprocs > 0
         assert len(self.running) == 1, "For now lets consider only one thread running"
         self._current = self.running[0]
-        self._function_abi = I386StdcallAbi(self.procs[0])
 
         #Install event forwarders
         for proc in self.procs:
             forward_signals(self, proc)
         
 
     @property
+    def _function_abi(self):
```

Comment:
I think the non-private method for this is [`invoke_model`](https://github.com/trailofbits/manticore/blob/520a9be47d4df0cdb4990c3b4f81318574bde8ed/manticore/platforms/platform.py#L32)